### ISSUE TITLE
tool use

### ISSUE THREAD

**Issue by wqlfff** [LG-AI-EXAONE/EXAONE-3.5#2]
Are you planning to add tool use (function calling) in the future?

**Comment by lgai-exaone** [comment]
Yes, we are planning to develop the function calling in the next version.

Thanks for your suggestion.
